src/components/text-editor/text-editor.tsx

```diff
@@ -1,34 +1,10 @@
-import {
-    Component,
-    Element,
-    Event,
-    EventEmitter,
-    Prop,
-    State,
-    h,
-} from '@stencil/core';
-import { EditorState } from 'prosemirror-state';
-import { EditorView } from 'prosemirror-view';
-import { Schema, DOMParser } from 'prosemirror-model';
-import { schema } from 'prosemirror-schema-basic';
-import { addListNodes } from 'prosemirror-schema-list';
-import { exampleSetup } from 'prosemirror-example-setup';
-import { MenuElement, MenuItem } from 'prosemirror-menu';
-import { buildFullMenu } from './menu/full-menu';
-import { PluginButton } from './menu/types';
-import { getFilteredMenu } from './menu/menu-filter';
-
+import { Component, Event, EventEmitter, Prop, h } from '@stencil/core';
+import { FormComponent } from '../form/form.types';
+import { EditorButton } from './menu/types';
 /**
- * This editor offers a rich text editing experience with markdown support,
- * in the sense that you can easily type markdown syntax and see the rendered
- * result as rich text in real-time. For instance, you can type `# Hello, world!`
- * and see it directly turning to a heading 1 (an `<h1>` HTML element).
- *
- * Naturally, you can use standard keyboard hotkeys such as <kbd>Ctrl</kbd> + <kbd>B</kbd>
- * to toggle bold text, <kbd>Ctrl</kbd> + <kbd>I</kbd> to toggle italic text, and so on.
+ * See `prosemirror-adapter` for more information
```

Comment:
The adapter component is private so a user will never be able to read anything for more information :smile: